locale/en/common.po

```diff
@@ -2105,6 +2105,13 @@ msgstr "Expand all"
 msgid "list.collapseAll"
 msgstr "Collapse all"
 
+msgid "list.expand"
+msgstr "Expand all"
+
+msgid "list.collapse"
+msgstr "Collapse all"
+
+
```

Comment:
```suggestion
```

Reply:
Actually intended to introduce new translation to have just "Collapse" and "Expand", so also fixed that.